packages/fxa-auth-client/.eslintrc.json

```diff
@@ -0,0 +1,22 @@
+{
+  "env": { "es6": true, "node": true },
+  "extends": ["plugin:fxa/recommended"],
+  "plugins": ["fxa"],
+  "rules": { "import/no-unresolved": "off" },
+  "root": true,
+  "ignorePatterns": ["dist"],
+  "overrides": [
+    {
+      "files": ["**/*.ts?(x)"],
+      "parser": "@typescript-eslint/parser",
+      "parserOptions": {
+        "ecmaVersion": 2018,
```

Comment:
In some packages we have this set to `2020` and others `2018`. Should they be consistent or does setting this to `2020` break things?

Reply:
Good catch, I'll try bumping them up to "2020" (to match fxa-auth-server) and see if linters gonna lint.

```sh
git grep -n '"ecmaVersion":' packages/**/.eslintrc* | cat

packages/fxa-auth-client/.eslintrc.json:13:        "ecmaVersion": 2018,
packages/fxa-auth-server/.eslintrc.json:8:    "ecmaVersion": "2020",
packages/fxa-event-broker/.eslintrc.json:12:        "ecmaVersion": 2018,
packages/fxa-graphql-api/.eslintrc.json:12:        "ecmaVersion": 2018,
packages/fxa-support-panel/.eslintrc.json:12:        "ecmaVersion": 2018,
```

Reply:
@LZoog Bumped everything to `2020` and re-ran the linters and everything passes.